code/_onclick/item_attack.dm

```diff
@@ -17,7 +17,7 @@
 
 // No comment
 /atom/proc/attackby(obj/item/W, mob/user, params)
-	return SendSignal(COMSIG_PARENT_ATTACKBY, W, user, params)
+	SendSignal(COMSIG_PARENT_ATTACKBY, W, user, params)

```

Comment:
The return was so that components could override the behaviour of attackby (which doesn't call afterattack if it returns TRUE). Why'd you remove it?

Reply:
Can we not change the structure of most used call chain every 2 weeks.

Reply:
Oh woops, I was pretty tired at the time and probably thought it did nothing.